packages/design-system/src/components/OcResource/OcResource.vue

```diff
@@ -273,6 +312,11 @@ export default defineComponent({
     }
   }
 }
+.condensed {
+  .oc-resource-thumbnail-status-badge {
+    bottom: 2px !important;
```

Comment:
if the table is condensed we need to move the badge up a few pixels to not overflow the row